<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataVisualizer</class>
 <widget class="QMainWindow" name="DataVisualizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1054</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Visualizer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1031</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="dataVisualizerLayout"/>
   </widget>
   <widget class="QComboBox" name="encodersSpinner">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="numDataPointsSpinBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="addDataPointsBtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QComboBox" name="activeEncodersSpinner">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="removeActiveEncoderBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>390</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeAllEncodersBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove All</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="liveDataCheckBox">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>380</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Live</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numPreviousDataPntsLineEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>390</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="setNumPreviousDataPntsBtn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>390</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
